src/app/app.component.ts

```diff
@@ -1,15 +1,16 @@
-import { Component } from '@angular/core';
```

Comment:
Can you add an example of using a component and nested one more popper inside it, please?